docs/upgradeFromD8toD9.md

```diff
@@ -0,0 +1,209 @@
+
+Run each line for D9 core and its base packages.
+
+```Shell
+docker exec -ti esmero-php bash -c "composer require drupal/core:^9 drupal/core-composer-scaffold:^9 drupal/core-project-message:^9 --update-with-dependencies --no-update"
+docker exec -ti esmero-php bash -c "composer require drupal/core-dev:^9 --update-with-dependencies --no-update"
+```
+
+Now we are going to update D8 Archipelago required and suggested modules into D9 requirements/compatible packages:
+
+```Shell
+docker exec -ti esmero-php bash -c "composer require drupal/form_mode_manager:^2.0@RC --update-with-dependencies --no-update"
+docker exec -ti esmero-php bash -c "composer require drupal/display_field_copy:^2.0 --update-with-dependencies --no-update"
+docker exec -ti esmero-php bash -c "composer require symfony/event-dispatcher:^4 --update-with-dependencies --no-update"
+docker exec -ti esmero-php bash -c "composer require symfony/yaml:^4  --update-with-dependencies --no-update"
+docker exec -ti esmero-php bash -c "composer require drupal/webform:^6  --update-with-dependencies --no-update"
+docker exec -ti esmero-php bash -c "composer require drupal/google_api_client:3.2 --update-with-dependencies --no-update"
+docker exec -ti esmero-php bash -c "composer require drupal/bootstrap_barrio:5.1 --update-with-dependencies --no-update"
+docker exec -ti esmero-php bash -c "composer require 'drupal/role_based_theme_switcher:^9' --update-with-dependencies --no-update"
+```
+
+*Note:* We are fixing `drupal/bootstrap_barrio` to an exact version (5.1) because 5.5 and later are now Bootstrap 5 and `archipelago_subtheme` is still Bootstrap 4. 
+
+We need to also remove legacy modules that are **no longer supported** by D9 or have been incorporated into `Core`. 
+
+```Shell
+docker exec -ti esmero-php bash -c "composer remove drupal/config_installer  --update-with-dependencies --no-update"
+docker exec -ti esmero-php bash -c "composer remove drupal/jsonapi_earlyrendering_workaround --no-update"
+docker exec -ti esmero-php bash -c "composer remove drupal/markdown --no-update"
+docker exec -ti esmero-php bash -c "composer remove drupal/module_missing_message_fixer --no-update "
+docker exec -ti esmero-php bash -c "composer remove drupal/role_theme_switcher --no-update"
+docker exec -ti esmero-php bash -c "composer remove drupal/key_value --no-update"
+docker exec -ti esmero-php bash -c "composer remove drupal/olivero --no-update"
+```
+
+And we uninstall those from Drupal too (IMPORTANT)
+
+```Shell
+docker exec -ti esmero-php bash -c " drush pm-uninstall module_missing_message_fixer"
+docker exec -ti esmero-php bash -c " drush pm-uninstall jsonapi_earlyrendering_workaround"
+docker exec -ti esmero-php bash -c " drush pm-uninstall markdown"
+docker exec -ti esmero-php bash -c " drush pm-uninstall role_theme_switcher"
```

Comment:
```suggestion
docker exec -ti esmero-php bash -c " drush pm-uninstall webprofiler"
```
Keep role_theme_switcher and remove webprofiler which has been merged with devel.